object_test.go

```diff
@@ -659,3 +661,91 @@ func TestIssue150_EmbedStructJavaDecode(t *testing.T) {
 
 	testJavaDecode(t, "customArgTypedFixed_Extends", dog)
 }
+
+type Mix struct {
+	A  int
+	B  string
+	CA time.Time
+	CB int64
+	CC string
+	CD []float64
+	D  map[string]interface{}
+}
+
+func (m Mix) JavaClassName() string {
+	return `test.mix`
+}
+
+func init() {
+	RegisterPOJO(new(Mix))
+}
+
+//
+// BenchmarkJsonEncode-8   	  217354	      4799 ns/op	     832 B/op	      15 allocs/op
```

Comment:
Placing benchmark informations into issue or readme may be better.

Reply:
maybe it's convernients for next coders ?

Reply:
Ok.